.taskcluster.yml

```diff
@@ -10,302 +10,305 @@ tasks:
               revision: cab4565345d0d0effa66f18fe91335dd6d744031
           trustDomain: mobile
       in:
-          $if: 'tasks_for in ["github-pull-request", "github-push", "action", "cron"]'
-          then:
-              $let:
-                  # Github events have this stuff in different places...
-                  ownerEmail:
+          $let:
+              # Github events have this stuff in different places...
+              ownerEmail:
+                  $if: 'tasks_for in ["cron", "action"]'
+                  then: '${tasks_for}@noreply.mozilla.org'
+                  else:
                       $if: 'tasks_for == "github-push"'
                       then: '${event.pusher.email}'
                       # Assume Pull Request
                       else:
                           $if: 'tasks_for == "github-pull-request"'
                           then: '${event.pull_request.user.login}@users.noreply.github.com'
                           else:
-                              $if: 'tasks_for in ["cron", "action"]'
-                              then: '${tasks_for}@noreply.mozilla.org'
-                  baseRepoUrl:
-                      $if: 'tasks_for == "github-push"'
-                      then: '${event.repository.html_url}'
-                      else:
-                          $if: 'tasks_for == "github-pull-request"'
-                          then: '${event.pull_request.base.repo.html_url}'
-                          else:
-                              $if: 'tasks_for in ["cron", "action"]'
-                              then: '${repository.url}'
-                  repoUrl:
-                      $if: 'tasks_for == "github-push"'
-                      then: '${event.repository.html_url}'
+                              $if: 'tasks_for == "github-release"'
```

Comment:
Reference-browser doesn't use releases so we've never added them in `.taskcluster.yml`. That said, I think we should make it as close as fenix'. This way, it's easier to track the differences. It was hard for me to make sure I didn't miss anything in my first patch.

The following changes in this `$let` statement are all about adding `github-releases`. The diff just looks bad to comprehend.

Reply:
I agree with these changes. I tried to do the same in app-services. It's better to have some unused variables than deal with identation errors and configs changes across these mobile repos 👍